ITEM_en.tsv

```diff
@@ -3229,16 +3229,16 @@ ITEM_20150317_003388	A great herb requested by Master Cleric
 ITEM_20150317_003389	Goddess Offering Pocket
 ITEM_20150317_003390	An offering pocket that the cleric uses to warship Goddess statue
 ITEM_20150317_003391	Balm
-ITEM_20150317_003392	A balm uses to repair Goddess staute. The feature is unusual and soft scent
+ITEM_20150317_003392	A balm uses to repair Goddess statue. The feature is unusual and soft scent
```

Comment:
ok.
This pull is just for fixing some typo.
There is 6 other line where Goddess Statue don't match case.
Will be fixed in another pull.